### PR TITLE
Add npmlist to Registry

### PR DESCRIPTION
**Pull request by hankchanocd**
[npmlist](https://github.com/hankchanocd/npmlist) - Fuzzy list npm module's dependencies.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please adhere to the contribution guidelines.

**Comment by sindresorhus** [comment]
The description doesn't explain very well what the tool actually does.

**Comment by sindresorhus** [comment]
I think it would be better to put it in the `CLI` section.

**Comment by hankchanocd** [comment]
My apology. I've added description to the comment above. 
You are right, `npmlist` suits better at `CLI` section since it focuses on fuzzy listing at command line.

Please let me know if there's anything else needed.

File: readme.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ You might also like [awesome-nodejs](https://github.com/sindresorhus/awesome-nod
 ### CLI
 
 - [zsh-better-npm-completion](https://github.com/lukechilds/zsh-better-npm-completion) - Better ZSH completion for npm.
+- [npmlist](https://github.com/hankchanocd/npmlist) - Fuzzy list npm module's dependencies.
```
Comment (sindresorhus) [inline readme.md]: The description is still not clear and according to your own readme, it works on any package property, not just dependencies.

**Comment by hankchanocd** [comment]
After a bit of reflection, I figured that I'd better off with a new and well-defined repo that has narrower focus. 

[npm-fzf](https://github.com/hankchanocd/npm-fzf) - Fuzzy search npm modules with [`fzf`](https://github.com/junegunn/fzf).

`npm-fzf` is the upgrade version of [`npmlist`](https://github.com/hankchanocd/npmlist/blob/master/README.md). I intend on publishing `npm-fzf` here rather than the old one, `npmlist`, which is now used as an API.

**Comment by sindresorhus** [comment]
I'm still lost on what I would use this for. What practical problems does it solve?

**Comment by sindresorhus** [comment]
Gonna pass on this.

**Pull request closed without merging**